src/modules/StartSection/components/ServicesList/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./services-list.scss";
+
+function ServicesList() {
+  return (
+    <ul className="search__services-list">
+      <li className="search__service-item">
```

Comment:
Options (li) should be dynamical - please, reside them into an array and loop through it

Reply:
done